### DISCUSSION TITLE
Il mio BT non ci arriva.

### DISCUSSION THREAD

**Discussion by pippo73** [indomus/forum#231] [GENERALE - Discussioni generiche]
Ciao a tutti.
vorrei acquistare i sensori per le piante della xiaomi. Se non sbagli usano tecnologia BT LE. Visto che le piante sono lontane dal mio raspberry, temo che il segnale possa non arrivarci. Secondo voi, come posso fare per "prolungare" il segnale?

**Comment by snakuzzo** [comment]
Così ... https://github.com/zewelor/bt-mqtt-gateway
o così... https://esphome.io/components/esp32_ble_tracker.html

**Reply by pippo73** [reply]
Scusa e come dovrebbe aiutarmi questo progetto?

**Reply by snakuzzo** [reply]
Hai letto almeno ?
Il primo è un gateway che installi su un device (es. Raspberry Pi Zero W) e ti fa da tramite tra i device BT ed il broker MQTT via Wifi
L'altro invece è un tracker BL che, se leggi, creato il firmware da caricare su un ESP32, ti permette di sfruttare quel device come tracker per poter recuperare i valori esposti dai sensori

**Reply by pippo73** [reply]
Li ho letti velocemente, ma come immaginavo prevede comunque un nuovo device da configurare e gestire...
Esistono invece soluzioni hardware?

**Reply by KC-inDomus** [reply]
un nuovo device è una soluzione hardware

**Reply by pippo73** [reply]
Non c'è peggior sordo di chi non vuole sentire 😒

**Reply by snakuzzo** [reply]
Beh! La risposta è stata data in base a ciò che hai chiesto
Ad ogni modo, fammi capire...vuoi una sorta di device che attacchi, NON configuri e che faccia tutto in automatico?
Se è questo che cerchi, non credo che esista.
Non credo vi sia in commercio un extender BT...ma magari mi sbaglio eh. Le soluzioni che ti ho linkato sono valide, ma se non ti piacciono, nulla ti vieta di spostare il Raspberry :)

**Comment by gforment** [comment]
Ciao, io ho acquistato dalla Cina uno xiaomi gateway v3 con il quale sono riuscito ad avvicinarmi ai sensori in giardino. Il gateway aggancia i mi flora che integro in HA con questa integrazione https://github.com/AlexxIT/XiaomiGateway3

**Reply by snakuzzo** [reply]
Non si accettano link ad altri siti. Leggete il regolamento

**Reply by pippo73** [reply]
Grazie @gforment , penso che alla fine optero' per questa soluzione. :-)
PS mi dispiace che ti abbia bacchettato @snakuzzo per il link ;-)

**Reply by gforment** [reply]
@snakuzzo ho letto il regolamento :) volevo mettere il link a github come i tuoi ad inizio thread ma devo aver sbagliato copia e incolla! Ho corretto
